ratelimits/source_redis.go

```diff
@@ -28,55 +25,42 @@ type RedisSource struct {
 
 // NewRedisSource returns a new Redis backed source using the provided
 // *redis.Ring client.
-func NewRedisSource(client *redis.Ring, timeout time.Duration, clk clock.Clock, stats prometheus.Registerer) *RedisSource {
-	if len(client.Options().Addrs) == 0 {
-		return nil
-	}
-	var addrs []string
-	for addr := range client.Options().Addrs {
-		addrs = append(addrs, addr)
-	}
-	labels := prometheus.Labels{
-		"addresses": strings.Join(addrs, ", "),
-		"user":      client.Options().Username,
-	}
-	stats.MustRegister(bredis.NewMetricsCollector(client, labels))
+func NewRedisSource(client *redis.Ring, clk clock.Clock, stats prometheus.Registerer) *RedisSource {
+	// Exponential buckets ranging from 0.0005s to 300ms.
+	buckets := prometheus.ExponentialBucketsRange(0.0005, 0.3, 8)
```

Comment:
If SRE is allowed to adjust the WFE read/write latency, this upper bound of 300ms may not make sense. The `ratelimits_spend_latency_seconds` histogram has an upper bound of 2s.